Formula/texlive.rb

```diff
@@ -0,0 +1,478 @@
+class Texlive < Formula
+  include Language::Python::Virtualenv
+  include Language::Python::Shebang
```

Comment:
Do you use this anywhere? Haven't spotted it.

Reply:
Whoops, I added that while testing ways to fix the Python scripts and forgot to remove it.  It's not needed.